README.md

```diff
@@ -26,7 +24,7 @@ Mountpoint for Amazon S3 is generally available! We're tracking future feature d
 ### Notices
 
 > [!IMPORTANT]
-> Mountpoint for Amazon S3 v1.4.0 released on January 26, 2024, contains an issue that causes intermittent read failures. Amazon S3 fixed the issue and released Mountpoint for Amazon S3 v1.4.1 on February 16, 2024.
+> Mountpoint for Amazon S3 v1.4.0, released on January 26, 2024, contains an [issue](https://github.com/awslabs/mountpoint-s3/pull/751) that causes intermittent read failures. We recommend that customers using v1.4.0 upgrade to v1.4.1 or later.
```

Comment:
minor thing, but is it the right link here? the issue is: https://github.com/awslabs/mountpoint-s3/issues/749, but I can agree with that the PR provide better description of the problem

Reply:
Yeah, I thought linking to the PR is more direct about what the issue is.